src/main.dart

```diff
@@ -1,5 +1,183 @@
-import 'core/note_app.dart';
+import 'dart:io';
 
+class Note {
+  late String title;
+  late String content;
+
+  Note({
+    required this.title,
+    required this.content,
+  });
+}
+
+//Manage operation of program (create,edit,delete,search,display).
+class NoteManager{
+  List <Note> notes=[];
+
+
+//Creates a new note and adds it to the list of notes.
+ void createNote(){
+  String title,content;
+  print("Enter the note title: ");
```

Comment:
It would have enhanced user-friendliness if you had incorporated a check to verify whether a title has already been used before proceeding with the creation of a new note.